examples/integrations/cdi/jpa/src/main/resources/META-INF/persistence.xml

```diff
@@ -17,10 +17,10 @@
 
 -->
 <persistence version="2.2"
```

Comment:
I think if our [JPA version is 3.0](https://github.com/oracle/helidon/blob/e2678dbb573ea4c7b8104f15dca1527369498aac/dependencies/pom.xml#L83) then all references to `2.2` here need to go to `3.0`.

Reply:
fixed